packages/remix-adapter/package.json

```diff
@@ -67,14 +67,12 @@
     "@remix-run/react": "^2.12.0",
     "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
-    "lambda-tester": "^4.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "tsup": "^8.0.2",
-    "vite": "^5.1.3"
+    "vite": "^5.4.11"
```

Comment:
I had to upgrade this to resolve some type issues caused by having multiple versions of vite in the workspace.